src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -178,28 +178,27 @@ public void testRegularExError() throws Exception {
         checkConfig.addProperty("tag", "@author");
         checkConfig.addProperty("tagFormat", "ABC");
         final String[] expected = {
-            "11: " + getCheckMessage(MSG_TAG_FORMAT, "@author", "ABC"),
+            "15: " + getCheckMessage(MSG_TAG_FORMAT, "@author", "ABC"),
         };
         verify(checkConfig, getPath("InputWriteTagExpressionError.java"), expected);
     }
 
     @Test
     public void testEnumsAndAnnotations() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(WriteTagCheck.class);
-        checkConfig.addProperty("tag", "@incomplete");
-        checkConfig.addProperty("tagFormat", ".*");
-        checkConfig.addProperty("severity", "ignore");
-        checkConfig.addProperty("tagSeverity", "error");
-        checkConfig.addProperty("tokens",
+        checkConfig.addAttribute("tag", "@incomplete");
```

Comment:
Hmm, can we stick to new name  of method in configuration ?
How can make sure we will not use deprecated method?

Reply:
> Hmm, can we stick to new name of method in configuration ?

Yes, I'm so sorry. I didn't notice that PR had been merged.

Reply:
> How can make sure we will not use deprecated method?

We can impose check for it in checkstyle_checks